config/prod/s3-synapse-sync.yaml

```diff
@@ -17,7 +17,8 @@ parameters:
       "htan-dcc-tma-tnp":{"SynapseProjectId":"syn22041595"},
       "htan-dcc-msk":{"SynapseProjectId":"syn23448901"},
       "htan-dcc-duke":{"SynapseProjectId":"syn23511961"},
-      "htan-dcc-tnp-sardana":{"SynapseProjectId":"syn24984270"}
+      "htan-dcc-tnp-sardana":{"SynapseProjectId":"syn24984270"},
+      "htan-dcc-hms":{"SynapseProjectId":"syn22123910"}
```

Comment:
@clarisse-lau: How did this bucket work before this?

Reply:
Xengie and I had set up S3 buckets for each center and attached them to their respective Synapse projects. For some centers e.g. Vanderbilt and CHOP that are uploading data on Synapse, these buckets are just used as Synapse storage locations. When/if collaborators request access to upload directly to a bucket, then we set it up with the s3-synapse-sync lambda

Reply:
Ah, I see! Thanks for the explanation!